docs/dg/dev/backend-development/audit-logs/audit-logs.md

```diff
@@ -0,0 +1,265 @@
+* [HowTo: Extend the log structure with additional data](/docs/pbc/all/miscellaneous/{{site.version}}/tutorials-and-howtos/how-to-extend-the-log-structure-with-additional-data.html).
+* [HowTo: Add a new audit log type](/docs/pbc/all/miscellaneous/{{site.version}}/tutorials-and-howtos/how-to-add-a-new-audit-log-type.md)
+---
+
+Audit logging is used in web applications for tracking user activities and detecting unauthorized access. It helps meet regulatory requirements by ensuring accountability and transparency. Additionally, audit logs help with troubleshooting by recording system events and user interactions, making it easier to identify and resolve issues.
+
+## AuditLoggerConfigPluginInterface
+
+Audit logging is enabled by various plugins that implement `Spryker\Shared\LogExtension\Dependency\Plugin\AuditLoggerConfigPluginInterface`. These plugins provide the necessary configuration for audit logging across different applications.
+
+`AuditLoggerConfigPluginInterface` defines the main configuration for audit loggers. Here are the main methods in this interface:
+
+<details>
+  <summary>AuditLoggerConfigPluginInterface</summary>
+
+```php
+<?php
+
+namespace Spryker\Shared\LogExtension\Dependency\Plugin;
+
+use Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer;
+
+/**
+ * Interface is used to provide configuration for audit logging.
+ */
+interface AuditLoggerConfigPluginInterface
+{
+    /**
+     * Specification:
+     * - Determines if the configuration is applicable based on the given criteria.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer
+     *
+     * @return bool
+     */
+    public function isApplicable(AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer): bool;
+
+    /**
+     * Specification:
+     * - Retrieves the name of the logging channel.
+     *
+     * @api
+     *
+     * @return string
+     */
+    public function getChannelName(): string;
+
+    /**
+     * Specification:
+     * - Retrieves the handlers for the logger.
+     *
+     * @api
+     *
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogHandlerPluginInterface>
+     */
+    public function getHandlers(): array;
+
+    /**
+     * Specification:
+     * - Retrieves the processors for the logger.
+     *
+     * @api
+     *
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogProcessorPluginInterface>
+     */
+    public function getProcessors(): array;
+}
+```
+
+</details>
+
+## Plugin configuration
+
+Each plugin supports one type of logs (channel). For instructions on adding audit log types, see [Add audit log types](/docs/pbc/all/miscellaneous/{{page.version}}/tutorials-and-howtos/how-to-add-a-new-audit-log-type.md).
+
+The configuration for these plugins is defined in `config/Shared/config_default.php`. Here's an example of how to register plugins for different applications:
```

Comment:
Could you please explain how to add new applications here?
Some customers have more applications than mentioned in this example